src/Connection/PgsqlConnection.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Pgsql\Connection;
+
+use PDO;
+use Yiisoft\Db\Connection\Connection;
+use Yiisoft\Db\Command\Command;
+use Yiisoft\Db\Exception\Exception;
+use Yiisoft\Db\Exception\InvalidConfigException;
+use Yiisoft\Db\Exception\NotSupportedException;
+use Yiisoft\Db\Pgsql\Schema\PgsqlSchema;
+
+/**
+ * Database connection class prefilled for PGSQL Server.
+ */
+final class PgsqlConnection extends Connection
```

Comment:
That looks a bit over-namespaced. I know that it is in line with `db` package namespaces but still... could it be `Yiisoft\Db\Pgsql\Connection`?

Reply:
Fixed.